six/modules/python/six/tests/runTests.py

```diff
@@ -65,96 +65,92 @@ def runCsmTests():
+    cphdPathname = os.path.join(cphdDir, os.listdir(cphdDir)[0])
 
-        if success != 0:
-            if os.path.exists('output.cphd'):
-                os.remove('output.cphd')
-            print("Error running test_round_trip")
-            return False
+    # Generate CPHD1.0 output file
+    success = subprocess.call([writeCphd,
+                               cphdPathname, 'output.cphd', cphdSchemas[0]],
+                              stdout=subprocess.PIPE)
 
-        # Check if CPHD1.0 input and output files match
-        cphdRunner = CppTestRunner(os.path.join(utils.installPath(), 'tests', 'cphd'))
-        result = cphdRunner.run('test_compare_cphd', cphdPathname, 'output.cphd', cphdSchemas[0])
+    print('Running test_round_trip')
 
-        # Remove output file after done
+    if success != 0:
         if os.path.exists('output.cphd'):
             os.remove('output.cphd')
+        print('Error running test_round_trip')
+        return False
+
+    # Check if CPHD1.0 input and output files match
+    cphdRunner = CppTestRunner(os.path.join(utils.installPath(), 'tests', 'cphd'))
+    result = cphdRunner.run('test_compare_cphd', cphdPathname, 'output.cphd', cphdSchemas[0])
+
+    # Remove output file after done
+    if os.path.exists('output.cphd'):
+        os.remove('output.cphd')
 
-        return result;
+    return result
 
 
 def run(sourceDir):
     # If we don't run this before setting the paths, we won't be testing
     # the right things
-
     sicdDir = os.path.join(sourceDir, 'SICD')
     siddDir = os.path.join(sourceDir, 'SIDD')
     cphdDir = os.path.join(sourceDir, 'CPHD1.0')
 
     if sourceDir != '':
-        os.environ["PATH"] = (os.environ["PATH"] + os.pathsep +
-                os.path.join(utils.installPath(), 'bin'))
+        os.environ['PATH'] = (os.environ['PATH'] + os.pathsep +
+                              os.path.join(utils.installPath(), 'bin'))
         cropSicds = utils.executableName('crop_sicd')
 
         sicdPathname = os.path.join(sicdDir, os.listdir(sicdDir)[0])
 
         success = subprocess.call([cropSicds,
-                '--start-row', '0', '--start-col', '0',
-                '--num-rows', '10', '--num-cols', '10',
-                sicdPathname, 'cropped.nitf'],
-                stdout=subprocess.PIPE)
+                                   '--start-row', '0', '--start-col', '0',
+                                   '--num-rows', '10', '--num-cols', '10',
+                                   sicdPathname, 'cropped.nitf'],
+                                  stdout=subprocess.PIPE)
 
-        print("Running crop_sicd")
+        print('Running crop_sicd')
         if os.path.exists('cropped.nitf'):
             os.remove('cropped.nitf')
         if success != 0:
-            print("Error running crop_sicd")
+            print('Error running crop_sicd')
             return False
 
     utils.setPaths()
 
     if platform.system() != 'SunOS':
         if makeRegressionFiles.run() == False:
```

Comment:
Not sure if this condition is safe, because .makeRegressionFiles.run() doesn't return a boolean type.
Should I just change the function to return a boolean?

Reply:
Ooh, good catch! Yeah, whatever makes sense.